### PR TITLE
Documentation update after iOS warnings fix in contrib/retina header

### PR DESCRIPTION
**Pull request by asmorkalov**

### CODE REVIEW

**Comment by asmorkalov** [comment]
@mshabunin Please look at it.

File: modules/contrib/doc/retina/index.rst [hunk 1]
```diff
@@ -309,7 +309,7 @@ Retina::setupIPLMagnoChannel
 Retina::setupOPLandIPLParvoChannel
 ++++++++++++++++++++++++++++++++++
 
-.. ocv:function:: void Retina::setupOPLandIPLParvoChannel(const bool colorMode = true, const bool normaliseOutput = true, const float photoreceptorsLocalAdaptationSensitivity = 0.7, const float photoreceptorsTemporalConstant = 0.5, const float photoreceptorsSpatialConstant = 0.53, const float horizontalCellsGain = 0, const float HcellsTemporalConstant = 1, const float HcellsSpatialConstant = 7, const float ganglionCellsSensitivity = 0.7 )
+.. ocv:function:: void Retina::setupOPLandIPLParvoChannel(const bool colorMode = true, const bool normaliseOutput = true, const float photoreceptorsLocalAdaptationSensitivity = 0.7f, const float photoreceptorsTemporalConstant = 0.5, const float photoreceptorsSpatialConstant = 0.53, const float horizontalCellsGain = 0, const float HcellsTemporalConstant = 1, const float HcellsSpatialConstant = 7, const float ganglionCellsSensitivity = 0.7 )
```
Comment (mshabunin) [inline modules/contrib/doc/retina/index.rst]: As I see in PR #3697 - other initializers have been changed too:
```.cpp
photoreceptorsTemporalConstant=0.5f
photoreceptorsSpatialConstant=0.53f
ganglionCellsSensitivity=0.7f
```

File: modules/contrib/doc/retina/index.rst [hunk 2]
```diff
@@ -293,7 +293,7 @@ Retina::write
 Retina::setupIPLMagnoChannel
 ++++++++++++++++++++++++++++
 
-.. ocv:function:: void Retina::setupIPLMagnoChannel(const bool normaliseOutput = true, const float parasolCells_beta = 0, const float parasolCells_tau = 0, const float parasolCells_k = 7, const float amacrinCellsTemporalCutFrequency = 1.2, const float V0CompressionParameter = 0.95, const float localAdaptintegration_tau = 0, const float localAdaptintegration_k = 7 )
+.. ocv:function:: void Retina::setupIPLMagnoChannel(const bool normaliseOutput = true, const float parasolCells_beta = 0, const float parasolCells_tau = 0, const float parasolCells_k = 7, const float amacrinCellsTemporalCutFrequency = 1.2f, const float V0CompressionParameter = 0.95, const float localAdaptintegration_tau = 0, const float localAdaptintegration_k = 7 )
```
Comment (mshabunin) [inline modules/contrib/doc/retina/index.rst]: One more initializer has been changed in PR #3697:
```.cpp
V0CompressionParameter=0.95f
```

**Comment by asmorkalov** [comment]
Good observation! I just relied on `check_docs2.py` script output. The remarks has been fixed.

**Comment by mshabunin** [comment]
:+1:

**Pull request merged**